ios-trivia-game/API/FirebaseClient.swift

```diff
@@ -98,7 +98,7 @@ class FirebaseClient {
     // get users in a game
     func getUsersInGameBy(roomId: String, complete: @escaping (FIRDataSnapshot) -> (), onError: ((Error?) -> ())?) {
         let path = "\(Constants.USER_IN_GAME_TABLE_NAME)"
-        ref.child(path).queryOrdered(byChild: "room_id").queryEqual(toValue: roomId).observe(.value, with: { (snapshot) in
+        ref.child(path).queryOrdered(byChild: "room_id").queryEqual(toValue: roomId).observeSingleEvent(of: .value, with: { (snapshot) in
```

Comment:
This one might be a good idea to keep as a listener? So that we can update the player count in real time. Thoughts?